mapswipe_workers/mapswipe_workers/firebase_to_postgres/archive_project.py

```diff
@@ -0,0 +1,41 @@
+"""
+Archive a project.
+
+Delete groups, tasks and results of a project in Firebase.
+Keep all informations in Postgres.
+"""
+
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+
+
+def archive_project(project_id: str) -> None:
+    """
+    Archive a project.
+
+    Call a function which deletes groups, tasks and results in Firebase.
+    Set archive = true for project in Postgres.
+    """
+    logger.info("Archive project with the id {0}".format(project_id))
+    delete_project_from_firebase(project_id)
+
+    fb_db = auth.firebaseDB()
+    ref = fb_db.reference("v2/projects/{0}/status".format(project_id))
+    ref.set({"archived"})
+
+    pg_db = auth.postgresDB()
+    sql_query = "UPDATE projects SET status = 'archived' WHERE project_id = {0}".format(
+        project_id
+    )
+    pg_db.query(sql_query)
+
+
+def delete_project_from_firebase(project_id: str) -> None:
```

Comment:
I find the function name a bit counter intuitive since only groups, tasks and results are deleted. maybe we can use `archive_project_in_firebase` or another term here

Reply:
Will remove the function and implement functionality in archive function.